src/SFA.DAS.Roatp.CourseManagement.Web.UnitTests/Controllers/StandardsControllerTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Roatp.CourseManagement.Web.Controllers;
+using SFA.DAS.Roatp.CourseManagement.Web.Infrastructure.ApiClients;
+using SFA.DAS.Roatp.CourseManagement.Web.Infrastructure.Authorization;
+using SFA.DAS.Roatp.CourseManagement.Web.Models.Standards;
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class StandardsControllerTests
+    {
+        private StandardsController _controller;
+        private Mock<IRoatpCourseManagementOuterApiClient> _outerApiClient;
+        private Mock<ILogger<StandardsController>> _logger;
+        private int _ukprn;
+        private StandardsListViewModel expectedModel;
+
+        [SetUp]
+        public void Before_each_test()
+        {
+            _outerApiClient = new Mock<IRoatpCourseManagementOuterApiClient>();
+            _logger = new Mock<ILogger<StandardsController>>();
+            _ukprn = 111;
+
+            var signInId = Guid.NewGuid();
+            var givenNames = "Test";
+            var familyName = "User";
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+            {
```

Comment:
Well done on tests, but we dont need all these claims setup, do we

Reply:
Thanks, claims setup removed not needed ones